src/app.rb

```diff
@@ -23,16 +23,14 @@ def initialize
   ACTIONS = {
     1 => :list_books,
     2 => :list_all_music_albums,
-    3 => :list_movies,
-    4 => :list_all_games,
-    5 => :list_all_genres,
-    6 => :list_labels,
-    7 => :list_all_authors,
-    8 => :list_sources,
-    9 => :add_a_book,
-    10 => :add_a_music_album,
-    11 => :add_a_movie,
-    12 => :add_a_game,
+    3 => :list_all_games,
+    4 => :list_all_genres,
+    5 => :list_labels,
+    6 => :list_all_authors,
+    7 => :add_a_book,
+    8 => :add_a_music_album,
+    9 => :add_a_movie,
```

Comment:
we dont have this as well

Reply:
Thanks for highlighting! I will resolve it.